README.md

```diff
@@ -4,27 +4,29 @@ Today we're going to build a clone of a social network (really similar to the bl
 
 As any other social network we're going to have different types of `Post`s. All these different types of Posts should inherit from the parent `Post` class:
 
-* `TextPost`: Just a simple text post. Should be constructed as `TextPost(text="Post Text").
-* `PicturePost`: A post containing text and the URL of a picture: Should be constructed as `PicturePost(text="Post Text").
+* `TextPost`: Just a simple text post. Should be constructed as `TextPost(text="Post Text")`.
+* `PicturePost`: A post containing text and the URL of a picture: Should be constructed as `PicturePost(text="Post Text", image_url="imgur.com/OAWTSJu")`.
 * `CheckInPost`: A post containing text and coordinates of the user's position. Should be constructed as `CheckInPost(text="Post Text", latitude="40.741895", longitude="-73.989308")`.
 
-Rmotrgram also has users. A user is a simple class that can be created as: `User(first_name, last_name, email)`.
+Rmotrgram also has users. A user is a simple class that can be created as: `User(first_name="John", last_name="Smith", email="johnsmith@rmotr.com")`. 
 
 ## Creating posts
 
-Posts are going to be created by users. Once we have our user created, we're going to use the `add_post` method from the user class. Example, to create a text post for our user John, we'll do something like:
+Posts are going to be created and then assigned a user. Once we have our user and post created, we're going to use the `add_post` method from the user class. `add_post` should add the post to the user's list of posts, and assign that user to the post. Example, to create a text post for our user John, we'll do something like:
 
 ```python
 john = User("John", "Lennon", "john@rmotr.com")
 text_post = TextPost("All you need is love!")
-text_post.user = None  # Important!
+text_post.user == None  # Important! Since the post has no user yet, the user attribute should be None.
```

Comment:
If this is an equality check, maybe it would be good to include the return value in a comment?